### PR TITLE
[BREAKINGCHANGE] Adjust props on TimeRangeControls to control each action

### PR DESCRIPTION
**Pull request by sjcobb**
Adjusts props on TimeRangeControls to allow embed users to pick and choose which buttons to show, they are all on by default. Make interval selector slightly less wide, although hard to override MUI padding here:
image

# Checklist

- [ ] Pull request has a descriptive title and context useful to a reviewer.
- [ ] Pull request title follows the `[<catalog_entry>] <commit message>` naming convention using one of the following `catalog_entry` values: `FEATURE`, `ENHANCEMENT`, `BUGFIX`, `BREAKINGCHANGE`, `IGNORE`.
- [ ] All commits have [DCO signoffs](https://github.com/probot/dco#how-it-works).

## UI Changes

- [ ] Changes that impact the UI include screenshots and/or screencasts of the relevant changes.
- [ ] Code follows the [UI guidelines](https://github.com/perses/perses/blob/main/ui/ui-guidelines.md).
- [ ] Visual tests are stable and unlikely to be flaky. See [Storybook](https://github.com/perses/perses/tree/main/ui/storybook#visual-tests) and [e2e](https://github.com/perses/perses/tree/main/ui/e2e#visual-tests) docs for more details. Common issues include:
  - Is the data inconsistent? You need to mock API requests.
  - Does the time change? You need to use consistent time values or mock time utilities.
  - Does it have loading states? You need to wait for loading to complete.

### CODE REVIEW

File: ui/components/src/RefreshIntervalPicker/RefreshIntervalPicker.tsx [hunk 1]
```diff
@@ -37,6 +38,12 @@ export function RefreshIntervalPicker(props: RefreshIntervalPickerProps) {
           inputProps={{
             'aria-label': `Select refresh interval. Currently set to ${formattedValue}`,
           }}
+          sx={{
+            maxWidth: 70,
+            '.MuiInputBase-inputSizeSmall': {
+              paddingLeft: 1,
+            },
+          }}
```
Comment (cndonovan) [inline ui/components/src/RefreshIntervalPicker/RefreshIntervalPicker.tsx:46]: The left padding looks OK to me right now (looking at my local perses with main brain) but you might be seeing something I'm not

File: ui/dashboards/src/components/TimeRangeControls/TimeRangeControls.tsx [hunk 2]
```diff
@@ -46,12 +46,17 @@ const DEFAULT_HEIGHT = '34px';
...
+  showRefreshInterval?: boolean;
 }
 
-export function TimeRangeControls({ heightPx, showRefresh = true }: TimeRangeControlsProps) {
+export function TimeRangeControls({
+  heightPx,
+  showTimeRangeSelector = true,
+  showRefreshButton = true,
+  showRefreshInterval = true,
+}: TimeRangeControlsProps) {
```
Comment (cndonovan) [inline ui/dashboards/src/components/TimeRangeControls/TimeRangeControls.tsx:59]: nice!

**Review by cndonovan — APPROVED** [review]
yay improvement!

I feel like if I have some time I might look into some kind of robust solution for the height of these buttons (the heights are not lining up). I think we're fighting with the MUI defaults for the heights of widgets

**Comment by saminzadeh** [comment]
Nice while we're at it, it would be great to be able to configure the time presets too:

```tsx
<TimeRangeControls timePresets={[{ value: '1h', label: "Last Hour" }, [{ value: '2h',  label: "Last 2 Hours" }]} />
```

**Review by saminzadeh — APPROVED** [review]

**Pull request merged**